custom_components/alexa_media/media_player.py

```diff
@@ -1430,12 +1439,8 @@ async def async_play_tts_cloud_say(self, public_url, media_id, **kwargs):
                 f"<audio src='{public_url}local/alexa_tts{output_file_name}' />"
             )
         else:
-            await self.async_send_tts(
-                "To send TTS, please set Announce=true. Music can't be played this way."
-            )
-            _LOGGER.warning(
-                "To send TTS, please set Announce=true. Music can't be played this way."
-            )
+            await self.async_send_tts("To send TTS, please set Announce=true.")
```

Comment:
This message too needs to just be a string.

Reply:
done